scheduler/feasible_test.go

```diff
@@ -202,7 +202,8 @@ func TestConstraintChecker(t *testing.T) {
 
 	nodes[0].Attributes["kernel.name"] = "freebsd"
```

Comment:
can you add test cases with constraints using the is_set/is_not_set operators?

Reply:
Oh yep, sorry - thought I'd already added those.